### ISSUE TITLE
⌨ Screen Reader Usability Issue with why the DEI form behaves differently than the others via-a-vis the profile

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5388]
## ❌ Current behaviour (issue)
Unlike the other sections, the section labeled "Diversity, equity and inclusion," does not close when I press the "Save and Close," button. The button does save, but the explanation and definitions of minority populations and the laws surrounding the equity act, still is visible.
### Category
Usability for AT users 
## Location
Diversity, Equity, and Inclusion form
### How to reproduce 
1.	Navigate to the DEI form within the application profile
2.	Navigate using your keyboard to the DEI section
3.	Choose an EE Group
4.	Hit save and go back. 
5.	Understand why you have been directed to the DEI page instead of the profile based on what you hear.
## ✔ Expected behaviour (fix)
The Save and go back steps are divided  between the from and EE Group Modals so that Screen reader user understands: 
1. That they need to save their EE group selection in the modal. 
2.  That they need to activate the go back to my profile link to continue building their profile. 
### Screen Captures of suggested changes
Modal: ![#3456 Capture.JPG](https://images.zenhubusercontent.com/625076b8f67c446dd188ffe5/f7f335a2-33e8-416a-823c-5704fc93228b)
DEI Form: 
![#3456 Capture 2.JPG](https://images.zenhubusercontent.com/625076b8f67c446dd188ffe5/fd28f777-f5f7-4ba3-a6ba-efded2019530)
### Acceptance Criteria
- [ ] DEI modal buttons are changed from "Save and go back" to  "Save and close"
- [ ] Combine existing paragraphs 1 and 2 together so they read:
> While the language around these categories is in need of updating, the Government of Canada will sometimes use these categories in hiring to make sure that it is meeting the aims of employment equity. These four groups are "women, Aboriginal peoples, persons with disabilities, and members of visible minorities."
- [ ] Create a new paragraph 2, with slightly different copy depending on of the user is in an application context:
> Activate the "Go back to my application profile" link to continue working on your (submission|profile).

**Comment by substrae** [comment]
This one seems a bit odd to me, because the current behaviour makes sense. "Save and close" refers to the dialog that pops up, not the page itself. I agree adding extra copy for SR users would be helpful here because the interaction differs slightly from other sections (there's a dialog). SR users are aware that they are inside a dialog at this point, so "Save and close" _should be_ helpful context given that it should indicate the dialog has closed after they've selected it.

**Comment by RM-1978** [comment]
Thanks @substrae for our chat on this! As we said, it might be clearer for the user if the text and the ARIA label for the  close button on the modal read:" Save and close,"  and we added go back to my application **profile** to the DEI page exit link. 

Thank you as well @substrae for your thoughts on the copy we might should update in the DEI page copy. You are right it could be condensed. Here is my recommended copy update: 

 **_The Employment Equity Act of Canada (1995) identifies four groups who have experienced systemic employment 
 barriers, and a number of obligations for the Government of Canada in addressing these barriers.

 While the language around these categories is in need of updating, the Government of Canada will sometimes use 
 these categories in hiring to make sure that it is meeting the aims of employment equity.  These four groups are "women, Aboriginal peoples, persons with disabilities, and members of visible minorities."

**This section is optional.** If you do not wish to self-identify on this platform, there is no obligation. To self-identify 
 and be considered for opportunities offered to these groups, select the buttons below to access information on each 
 group and confirm your affiliation. 

 Activate the "Go back to my application profile" link to continue working on you _submission."_**

**Comment by petertgiles** [comment]
Hi Rosalie, I've tried to synthesize your conversation with Josh into updated AC for this issue.  Is it accurate and ready to be worked on now?

**Comment by RM-1978** [comment]
Good to go @petertgiles

**Comment by esizer** [comment]
So, it looks like this dialog has changed since we migrated to the new profile with inline forms. However, I think a new issue has arose.

I still need to enter a signature and "sign" the form to exclude Indigenous identity which was a little confusing.

Would we want to make the signature field conditional based on the user checking an Indigenous identity?

Also, I may be wrong but I think we can close this now as partially solved.

![Screenshot 2023-08-03 125421](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/6f1f61e7-646a-49e2-a0d2-3e4d76f12cf7)

**Comment by RM-1978** [comment]
@esizer, we can close it as partially solved, but we should open another ticket for the part that is not resolved.
